tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -292,7 +289,7 @@ CMD ["/bin/bash"]
 USER 1000
 EXPOSE 8052
 
-ENTRYPOINT ["/usr/bin/tini", "--"]
+ENTRYPOINT ["dumb-init", "--"]
```

Comment:
We should use the full path here: `/usr/bin/dumb-init`

Reply:
@rooftopcellist Why tho?

Reply:
I just noticed we had previously done it with tini and @pabelanger   did it with dumb-init in the Dockerfile for python-base (used for the EE containers).
I was thinking it was best practice because installing it a different way (like via pip instead of the rpm) might cause it to favor the wrong path. 
 
But in this case, we control how it is installed, and will only be installing with pip upstream and rpm downstream... So it's a non issue.